modules/conversion-center/classes/render/core-render.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Classes\Render;
+
+use Elementor\Utils;
+
+/**
+ * Class Core_Render.
+ *
+ * This class handles the rendering of the Link In Bio widget for the core version.
+ *
+ * @since 3.23.0
+ */
+class Core_Render extends Render_Base {
+
+	public function render( \Elementor\Modules\ConversionCenter\Widgets\Link_In_Bio $widget ): void {
+		$settings = $widget->get_settings_for_display();
+
+		/**
+		 * Map args to internals
+		 */
+
+		//  Identity Image
+		$identity_image_props_style = $settings['identity_image_style'] ?? 'profile';
+		$identity_image_value       = $settings['identity_image'] ?? [];
+
+		//  Bio Heading
+		$bio_heading_output    = '';
+		$bio_heading_props_tag = $settings['bio_heading_tag'] ?? 'h2';
+		$bio_heading_value     = $settings['bio_heading'] ?? '';
+
+		//  Bio Title
+		$bio_title_output    = '';
+		$bio_title_props_tag = $settings['bio_title_tag'] ?? 'h2';
+		$bio_title_value     = $settings['bio_title'] ?? '';
+
+		//  Bio Description
+		$bio_description_output = '';
+		$bio_description_value  = $settings['bio_description'] ?? '';
+
+		// Icons
+		$icons_value = $settings['icon'] ?? [];
+
+		// CTAs
+		$ctas_value = $settings['cta_link'] ?? [];
+
+		/**
+		 * Checks
+		 */
+
+		$has_bio_description = ! empty( $bio_description_value );
+		$has_bio_heading     = ! empty( $bio_heading_value );
```

Comment:
@louiswol94 Can you please confirm?

```suggestion
		$has_bio_heading = ! empty( $bio_heading_value );
```

Reply:
Answered above

Reply:
@louiswol94 @hein-obox so for now I have left the spacing untouched, could we think about adding the spacing rules to phpcs?In this way they are enforced by phpcbf automatically. If you prefer for me to use the other spacing without putting it in phpcs that's ok

Reply:
@nicoladj77 Sorry, I just thought about it again, can you remove these spacings please. We will try to add it to phpcs at a later stage. Thanks!